src/mlpack/core/dists/gamma_distribution_impl.hpp

```diff
@@ -10,25 +10,28 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
+#ifndef _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_IMPL_HPP
+#define _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_IMPL_HPP
```

Comment:
Is _ at beginning intentional?
Same in `gamma_distribution.hpp`

Reply:
No, probably someone missed it in the past,  you can put whatever you want in `#define`, just remove it